addOns/requester/src/main/java/org/zaproxy/zap/extension/requester/ExtensionRequester.java

```diff
@@ -67,9 +92,30 @@ public void hook(ExtensionHook extensionHook) {
             menu.addPopupMenuItem(getPopupMsgMenuRequester());
             // ToolsMenuItem
             menu.addToolsMenuItem(new ToolsMenuItemRequester(this));
+
+            if (isDeprecated(org.zaproxy.zap.extension.stdmenus.PopupMenuResendMessage.class)) {
+                resendDialog =
+                        new ManualHttpRequestEditorDialog(true, "resend", "ui.dialogs.manreq");
```

Comment:
Instead of providing the Manual Request Editor from this add-on, can we have the `Open/Resend with Request Editor` option do the same thing as the `Open in Requester Tab...` option? Then we can remove the Manual Request Editor from core completely.

Reply:
There are other add-ons using the core manual request editor (and providing more functionality than what this add-on currently can), the plan is to move the core functionality to this add-on and change those add-ons to use it. (And in time provide a dialogue with the multiple tabs, as that's been asked before.)

I kept the same position as in core, happy to change it though.